### PR TITLE
Add oidc params for token refresh

### PR DESCRIPTION
**Pull request by goseind**
Adding:

```
oidc_scope = openid profile offline_access
oidc_issuer = escape
auth_oidc_refresh_active true
auth_oidc_refresh_before_exp 20
```

To `rucio.cfg` for the user to try automatic token refresh by rucio itself (which is, however, independent of the plugin, which will still need the other token refresh mechanism).

### CODE REVIEW

File: infrastructure/cluster/flux-v2/jhub/jhub-release.yaml [hunk 1]
```diff
@@ -125,6 +125,10 @@ spec:
                 echo "account = $JUPYTERHUB_USER" >> /opt/rucio/etc/rucio.cfg;
                 echo "auth_type = oidc" >> /opt/rucio/etc/rucio.cfg;
                 echo "oidc_audience = rucio" >> /opt/rucio/etc/rucio.cfg;
+                echo "oidc_scope = openid profile offline_access" >> /opt/rucio/etc/rucio.cfg;
+                echo "oidc_issuer = escape" >> /opt/rucio/etc/rucio.cfg;
+                echo "auth_oidc_refresh_active true" >> /opt/rucio/etc/rucio.cfg;
+                echo "auth_oidc_refresh_before_exp 20" >> /opt/rucio/etc/rucio.cfg;
```
Comment (garciagenrique) [inline infrastructure/cluster/flux-v2/jhub/jhub-release.yaml:131]: The syntax of lines 130 and 131 are not exactly the same (there is no `=` sign). Is this correct or just an oversight ?

Comment (garciagenrique) [inline infrastructure/cluster/flux-v2/jhub/jhub-release.yaml:131]: I also have in my config the following line
`oidc_polling = true`

Comment (goseind) [reply infrastructure/cluster/flux-v2/jhub/jhub-release.yaml:131]: According to the [rucio](https://rucio.cern.ch/documentation/user/using_the_client) docs this is the correct syntax.

Comment (goseind) [reply infrastructure/cluster/flux-v2/jhub/jhub-release.yaml:131]: I added it.

**Comment by goseind** [comment]
Turn out this is an error in the rucio docs:

```
configparser.ParsingError: Source contains parsing errors: '/opt/rucio/etc/rucio.cfg'
        [line 10]: 'auth_oidc_refresh_active true\n'
        [line 11]: 'auth_oidc_refresh_before_exp 20\n'
```

But it does not solve the problem as expected.
pr for correction #187

**Pull request merged**